lib/module.js

```diff
@@ -81,17 +81,12 @@ module.exports = function sentry (moduleOptions) {
 
   const initializationRequired = options.initialize && options.dsn
 
-  if (options.disabled) {
-    logger.info('Errors will not be logged because the disable option has been set')
-    return
```

Comment:
Maybe keep the log, just without return.

Reply:
I added     logger.info('Sentry client side errors will not be logged because the disable option has been set')
and     logger.info('Sentry server side errors will not be logged because the disable option has been set')
instead further down the code :)